cmd/testing/tls_test.go

```diff
@@ -0,0 +1,44 @@
+package cmdtesting
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func TestTLSSuite(t *testing.T) {
+	suite.Run(t, new(TLSSuite))
+}
+
+type TLSSuite struct {
+	BaseTLSSuite
+}
+
+func (s *TLSSuite) TestTLSflagWithSelfSignedCertificate() {
+	_, out, err := s.ExecuteTestCobraCommand("list", "--tls",
+		"--api-host", s.Host,
+		"--api-port", fmt.Sprint(s.Port))
+
+	s.Require().NoError(err, "failed to execute Cobra Command")
+	//TO DO: Change ExecuteTestCobraCommand to pass the certificate verification error
```

Comment:
Better to just add a ticket to the issue tracker and remove the TODO as they just clutter up the code.